src/Common/src/Interop/OSX/Interop.CoreFoundation.CFData.cs

```diff
@@ -20,30 +20,21 @@ internal static partial class CoreFoundation
         [DllImport(Libraries.CoreFoundationLibrary)]
         private static extern CFIndex CFDataGetLength(SafeCFDataHandle cfData);
 
+        internal static unsafe Span<byte> CFDataDangerousGetSpan(SafeCFDataHandle cfData)
+        {
+            long length = CFDataGetLength(cfData).ToInt64();
+            byte* dataBytes = CFDataGetBytePtr(cfData);
+            return new Span<byte>(dataBytes, checked((int)length));
```

Comment:
I assume there's no possibility of an overflow here so large that it wraps around to something valid?

Reply:
The only potentially weird values I can think of are between 2^63 and 2^64 (interpreted as negative length), if it was epsilon-close to 2^64 then that would be a legal negative int, but then the span ctor would throw. (And that would require hardware with more than 53 (or whatever fifty-something it is) bit addressing).

If there's another value range that's worrisome, I'm open to education, but I think `checked` is taking care of the rest of it.

Reply:
I totally missed the "checked".